core/sql/ustat/hs_cli.cpp

```diff
@@ -5442,12 +5427,17 @@ NAString HSSample::getTempTablePartitionInfo(NABoolean unpartitionedSample,
 //
 void HSSample::addTruncatedSelectList(NAString & qry)
   {
+    bool first = true;
     for (Lng32 i = 0; i < objDef->getNumCols(); i++)
       {
-        if (i)
-          qry += ", ";
+        if (!ComTrafReservedColName(*objDef->getColInfo(i).colname))
```

Comment:
If possible, it would be better to check whether the column is a system column, I assume that is the intent here? There is an undocumented option to give the divisioning columns regular, non-reserved names.